src/harness/reference_models/geo/utils.py

```diff
@@ -254,11 +254,30 @@ def PolygonsAlmostEqual(poly_ref, poly, tol_perc=10):
     True if the two polygons are equal (within the tolerance), False otherwise.
   """
   if isinstance(poly_ref, dict):
-    poly_ref = GeoJsonToShapelyGeometry(poly1)
+    poly_ref = GeoJsonToShapelyGeometry(poly_ref)
   if isinstance(poly, dict):
     poly = GeoJsonToShapelyGeometry(poly)
 
   union_polys = poly_ref.union(poly)
   intersection_polys = poly_ref.intersection(poly)
   return ((GeometryArea(union_polys) - GeometryArea(intersection_polys))
           < tol_perc/100. * GeometryArea(poly_ref))
+
+
+def hasPolygonCorrectGeoJsonWinding(polygon):
+  """
+  Args:
+    polygon: the reference GeoJSON geometry of  polygon object.
+
+   Returns: boolean set to true if the polygon has a correct GeoJson Winding and false otherwise.
+  """
+  shapely_polygon = GeoJsonToShapelyGeometry(polygon);
+  print(shapely_polygon.exterior)
+  if not shapely_polygon.exterior.is_ccw:
+    return False
+  #check the holes of the polygons
+  for ring in shapely_polygon.interiors:
+    if ring.is_ccw:
+      return False
+  return True;
```

Comment:
No need for the semicolon.

Reply:
thank you, it is fixed (sorry culture of C languages)